apps/fabric-website/src/pages/Overviews/ResourcesPage/docs/default/ResourcesDeveloperResources.md

```diff
@@ -7,6 +7,14 @@ Get started with React and learn how to build your first projects.
   <li class="mdut--full">[Fabric React tutorial](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Getting-Started-with-UI-Fabric)</li>
 </ul>
 
+### Documentation
+
+In addition to the API documentation and control examples found in the [Controls section](#/controls/web) of this website, you can find a full collection of API references on the docs.microsoft.com portal.
```

Comment:
Should we consider hyperlinking to the portal also?

Reply:
I consider that not... don't wanna confuse people with extra links that go to a general place from where they don't have a clear entry point to our Fabric documentation.

Reply:
@KevinTCoughlin do you mean adding a link to the Fabric docs to `docs.microsoft.com`? I was wondering that too, but then it's linked right below, so was thinking it might be duplicative.